### DISCUSSION TITLE
Add a mobgriefing rule for piglins

### DISCUSSION THREAD

**Discussion by Tzpi89** [PurpurMC/Purpur#508] [Feature Requests]
In vanilla minecraft Piglins picking up gold is considered mobs griefing because when mob griefing is turned off they can't do that anymore so it would be nice to have a toggle for that in the config

**Comment by akdukaan** [comment]
It looks like there is a config option already. Although I just tried it out, and it doesn't seem to work. Piglin can pick up gold ingots only when mobGriefing is enabled. The bypass-mob-griefing config option doesn't seem to have an affect for this mob.
